src/utils/_mixins.scss

```diff
@@ -19,3 +19,9 @@
     cursor: not-allowed;
     opacity: 0.6;
 }
+
+%is-focused {
+    border-radius: 4px;
```

Comment:
I am wondering if it would be better to remove `border-radius` and use the radius of the item instead. It will prevent overriding this style in some cases (eg: PrimaryNavItem)